src/pages/docs/matchmaker/guides/custom-games.mdx

```diff
@@ -6,25 +6,37 @@ the configurations available with custom lobbies.
 
 ## Creation
 
-To create custom lobbies, you must first enable them in the version config:
+To create custom lobbies, you must first enable them in your version config:
 
 <CodeGroup title='rivet.yaml'>
-  ```yaml matchmaker: game_modes: default: actions: create: enabled: true # Public custom lobbies will be
-  listed in the response to `lobbies.list`, private custom # lobbies will not enable_public: true
-  enable_private: true max_lobbies_per_identity: 5 # Optional ```
+  ```yaml
+  # ...
+
+  matchmaker:
+    game_modes:
+      default:
+        actions:
+          create:
+            enabled: true
+            # Public custom lobbies will be listed in the response to `lobbies.list`, private custom
+            # lobbies will not
+            enable_public: true
+            enable_private: true
+            max_lobbies_per_identity: 5 # Optional
+  ```
+
 </CodeGroup>
 
 Now users can create a new custom lobby on-demand like so:
 
 <CodeGroup title='Request' tag='POST' label='https://matchmaker.api.rivet.gg/v1/lobbies/create'>
-
-```bash {{ "title": "cURL" }}
-curl
--X POST \
--H "Content-Type: application/json" \
--d "{ \"game_mode\": \"default\", \"region\": \"atl\", \"publicity\": \"private\" }" \
-'https://matchmaker.api.rivet.gg/v1/lobbies/create'
-```
+  ```bash {{ "title": "cURL" }}
+  curl
+    -X POST \
+    -H "Content-Type: application/json" \
+    -d "{ \"game_mode\": \"default\", \"region\": \"atl\", \"publicity\": \"private\" }" \
+    'https://matchmaker.api.rivet.gg/v1/lobbies/create'
```

Comment:
curl is _scary_, can we do typescript?

Reply:
the typescript code is right below this, its in a code group